draft-ietf-taps-arch.md

```diff
@@ -298,15 +298,15 @@ The following diagram summarizes the top-level concepts in the architecture and
 
 Fundamentally, a Transport Services API needs to provide connection objects ({{objects}}) that allow applications to establish communication, and then send and receive data. These could be exposed as handles or referenced objects, depending on the language.
 
-Beyond the connection objects, there are several high-level groups of actions that any Transport Services API implementing this specification MUST provide:
+Beyond the connection objects, there are several high-level groups of actions that any Transport Services API needs to provide:
 
-* Pre-Establishment ({{preestablishment}}) encompasses the properties that an application can pass to describe its intent, requirements, prohibitions, and preferences for its networking operations. For any system that provides generic Transport Services, these properties SHOULD apply to multiple transport protocols. Properties specified during Pre-Establishment can have a large impact on the rest of the interface: they modify how establishment occurs, they influence the expectations around data transfer, and they determine the set of events that will be supported.
```

Comment:
Disagree on this: I think this is an archtiectural requirement:

It is RECOMMENDED that any system that provides generic Transport Services,  SHOULD define properties common to multiple transport protocols.